### PR TITLE
Update managed project/package to VS2017

### PR DESCRIPTION
**Pull request by sgarske**
Story: https://github.com/TechSmith/CamtasiaWin/issues/9202

### CODE REVIEW

**Comment by ajorians** [comment]
Looks good to me! 👍 :smiley:

File: Vimeo-DotNet/VimeoAPI/VimeoAPI.csproj [hunk 1]
```diff
@@ -32,6 +32,7 @@
     <WarningLevel>4</WarningLevel>
     <Prefer32Bit>false</Prefer32Bit>
     <LangVersion>latest</LangVersion>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
```
Comment (ajorians) [inline Vimeo-DotNet/VimeoAPI/VimeoAPI.csproj]: Did we want this when your configuration is `Release` as well:
image

:smiley:

**Comment by ajorians** [comment]
Still good for me! 👍 :smiley:

File: Vimeo-DotNet/VimeoAPI/VimeoAPI.csproj (hunk 1 above)
Comment (sgarske) [reply Vimeo-DotNet/VimeoAPI/VimeoAPI.csproj]: Yes, changed it

**Pull request merged**